src/styles/application.css

```diff
@@ -10,6 +10,12 @@
   }
 }
 
+@media (min-width: 90rem) {
+  .sto-app {
+    width: 75rem; /* White space on the right on huge screen */
```

Comment:
I was wondering if it was not better to use `max-width` instead of `width` here? I would say no with the media query here, but what do you think @GoOz ?

Reply:
Just FYI if you don't want to bothet with rem calculations you can use the mixin `rem(1200)`.

As for the MQ, since it's a `max-width` now, I think it became useless, indeed.

Reply:
Not in the store since we don't use stylus here^^

Reply:
oh right… 😶